pkg/api/bzz.go

```diff
@@ -452,7 +456,10 @@ func (s *Service) serveManifestEntry(
 // downloadHandler contains common logic for dowloading Swarm file from API
 func (s *Service) downloadHandler(logger log.Logger, w http.ResponseWriter, r *http.Request, reference swarm.Address, additionalHeaders http.Header, etag bool) {
 	headers := struct {
-		Cache *bool `map:"Swarm-Cache"`
+		Cache                 *bool           `map:"Swarm-Cache"`
+		Strategy              getter.Strategy `map:"Swarm-Redundancy-Strategy"`
+		FallbackMode          bool            `map:"Swarm-Redundancy-Fallback-Mode"`
+		ChunkRetrievalTimeout time.Duration   `map:"Swarm-Chunk-Retrieval-Timeout"`
```

Comment:
what is the purpose of this?

Reply:
@istae Configuring this counts as an advanced dev option. 
By setting chunk retrieval timeout to a suffienctly short interval one can effectively render some slow chunks not retrievable and thus 'simulate' chunk loss even in environments of perfect availability. 
This is especially useful since it allows for client side trigger of redundancy-based recovery of data implemented in this PR.